uamqp/message.py

```diff
@@ -116,6 +120,42 @@ def __init__(self,
             self._annotations = annotations
             self._header = header
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        state["_body"] = list(state["_body"].data) if state["_body"].data else ""
```

Comment:
also wondering in the state dict, should we have key with the underscore which indicates it's a private thing?
or we should just have `state['body']=...`.

from my perspective, pickle support is part of the public api surface, and by setting `_<variable>` then we're leaking kinda private information.

Reply:
that makes sense, and I'm assuming that should be the case for all the private attributes?